virus/virTemplate.md

```diff
@@ -1,40 +1,81 @@
-# {Q1A Paper Title}
-written by: [yourname](https://github.com/yourname *optional link*)
+# Possible novel rhabdovirus whose anomalous structural genes will g-"rhabdo" your attention
+written by: [James Shi](https://github.com/jamesshi567)
 
-### {Q1B Virus Name}
-*{Q1B, etiology of virus name}*
+### Apiovirus anomala
+*We believe based on current data that this virus has the closest relation to the Apis rhabdovirus. However, it remains unclassified beyond the family leve, as many rhabdoviruses are due to their high diversity. Thus, there is inherently some uncertainty in the assignment of a binomial name for this virus. I assigned a genus mimicking how "Apis" refers to bees, a highly suspected host for this virus. The second term anomala refers to the unexpected genomic organization of this virus.*
```

Comment:
include query coverage and percentage identity between these two sequences in the opening paragraph where making this comparison.